### PR TITLE
Testing environment

### PR DESCRIPTION
**Pull request by ruflin**
Testing environment with a Mock Beat to also create integration tests for libbeat.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,10 +5,20 @@ go:
...
+
 services:
   - redis-server
   - elasticsearch
 
+addons:
+  apt:
+    packages:
+      - python-virtualenv
+      - geoip-database
```
Comment (andrewkroh) [inline .travis.yml]: Not a really a new issue, but I am pretty sure the geoip-database package does not contain the GeoIP city database that libbeat requires to actually provide coordinates. Any queries to this DB return a named place, but not coordinates.

**Comment by andrewkroh** [comment]
LGTM.

File: .travis.yml (hunk 1 above)
Comment (ruflin) [inline .travis.yml]: Is this dependency even needed to run the tests?

**Comment by ruflin** [comment]
@urso Good points. We should discuss this further. Also if we want to directly apply it to this pull request or refactor it in a second stage.

**Pull request merged**